src/ui/layout/RecentObjectsListItem.vue

```diff
@@ -99,7 +99,7 @@ export default {
     }
   },
   mounted() {
-    this.previewAction = new PreviewAction(this.openmct);
+    this.previewAction = this.openmct.actions.getAction('preview');
```

Comment:
Get the action from the registry instead of creating a new one

Reply:
Oh man we do `import PreviewAction`, `new PreviewAction` all over the place. Can we replace all of them?

Reply:
Yes! I realize now I missed quite a few of these